web-client/src/modules/requests/components/RequestPostItem.tsx

```diff
@@ -1,36 +1,105 @@
-import { Button, Col, Row } from 'antd';
+import { /* TODO:(es) warning message Button, */ Col, Row } from 'antd';
 import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
 import { StepBackButton, StepForwardButton } from 'src/components/Buttons';
 import { Offer } from 'src/models/offers';
 import { Request } from 'src/models/requests';
+import { TimelineViewLocation } from 'src/modules/timeline/constants';
 import styled, { keyframes } from 'styled-components';
 
 import { COLORS } from '../../../theme/colors';
 import avgRating from '../assets/pinAverageRating.svg';
 import defaultUserPic from '../assets/role_pin.png';
-import warningSign from '../assets/warningExclamation.svg';
+

```

Comment:
This needs to be reimplemented and put in a function, as below.  The code needs work on clarity.  Hardcoding values is not a good practice, these should be constants.  The variables it is using are no longer relevant.

Reply:
Maybe this could wait to 3.1?  Data model changes might effect this.  It does not seem like critical functionality.  Bettter would be to display number of days since last response.